x-pack/test/performance/tests/playwright/ecommerce_dashboard.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import Url from 'url';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ecommerceDashboard({ getService }: FtrProviderContext) {
+  describe('ecommerce_dashboard', () => {
+    const config = getService('config');
+    const playwright = getService('playwright');
+    const logger = getService('log');
+
+    const { step } = playwright.makePage('ecommerce_dashboard', { autoLogin: true });
+
+    step('Go to Sample Data Page', async ({ page }) => {
```

Comment:
Do we really want to track the performance of all these intermediate steps? Or it's just an example? I'm asking because we are interested in the last step only.

Reply:
You are right but we thought that it would be helpful to logically organize the steps it's just for debugging purposes.